content/benefits-pay-perks/benefits-perks/spending-company-money.md

```diff
@@ -42,13 +42,16 @@ The estimated spend for interns is **$925\*** and includes only the electronic i
 
 ## Computers
 
-Once your contract is signed, you will need to get your Sourcegraph computer. Computers are considered property of the company. Please see below how to get your computer and which specs to get depending on your role.
+Computers are considered property of the company and all teammates must work from a company owned laptop. Please see below how to get your computer and which specs to get depending on your role.
 
-**US/Canada/EU-based teammates** need to place their order via [this form](https://forms.gle/1PVxXwLJ9WFJvref9) and our Tech Ops team will purchase them through CDW US/CA or our partner in the UK. (Please note there is nothing we can do about wait times and will update you when the order has been placed and when we have any more information about shipping dates).
+**US/Canada/EU-based teammates** Please place an order via [this form](https://forms.gle/1PVxXwLJ9WFJvref9) and our Tech Ops team will purchase them through CDW US/CA or our partner in the UK. (Please note there is nothing we can do about wait times and will update you when the order has been placed and when we have any more information about shipping dates).
```

Comment:
I think EU based limits it to European Union. Could we be more specific? For instance, Ukraine and UK are not part of the EU but we do send laptops there. Maybe a list of countries would help? Or simply state "Europe based (except Russia) teammates"